pesayetu/static/css/pesayetu.scss

```diff
@@ -0,0 +1,369 @@
+  }
+
+  // Remove attribution without having to change entire body block
+  section:nth-child(2) {
+    display: none;
+  }
+}
+
+// PAGES
+
+// Homepage
+
+#homepage .hero {
+  background-image: url('/static/img/banners/global.jpg');
+  background-size: cover;
+  background-position: center;
+  min-height: 300px;
+  padding: 150px 0;
+
+  select, a {
+    max-width: 100%;
+    width: 320px;
+    height: auto;
+    display: inline-block;
+    padding: 13px 20px 12px;
+    background-color: $lightblue;
+    text-transform: uppercase;
+    text-align: left;
+    color: #fff;
+    border: none;
+    font-size: 16px;
+    font-family: $font-poppins;
+    border-radius: 5px;
+    box-shadow: 0 0 20px #000;
+    cursor: pointer;
+    i {
+      float: right;
+      margin-top: 2px;
+    }
+    &:hover {
+      background-color: $red;
+    }
+  }
+  #locate-me {
+    padding-right: 13px;
+  }
+}
+
+.showcase {
+  padding: 50px 0;
+  .column-third {
+    .img {
+      height: 200px;
+      width: 100%;
+      max-width: 300px;
+      margin: 10px auto 20px;
+      background-size: cover;
+      background-repeat: no-repeat;
+      background-position: center center;
+    }
+  }
+
+  &.partners {
+    img {
+      max-width: 250px;
+      max-height: 75px;
+      margin: 10px;
+      &:hover {
+        opacity: 0.7;
+      }
+    }
+  }
+}
+
+.btn {
+  text-decoration: none;
+  &.btn-lg {
+    font-size: 1.2em;
+    padding: 10px 20px 12px;
+  }
+  &.bg-ocean-green:hover {
+    background-color: $red;
+    color: #fff;
+  }
+  &.bg-bottle-green:hover {
+    background-color: $red;
+    color: #fff;
+  }
+}
+
+
+// Select2 Stuff
+
+.select2-container {
+  .select2-choice {
+    div {
+      background-color: $red !important;
+      border: none !important;
+      background-image: url('/static/img/arrow-down-white.png') !important;
+      background-size: 24px !important;
+      background-repeat: no-repeat !important;
+      background-position: left center !important;
+      width: 34px !important;
+      b {
+        display: none !important;
+      }
+    }
+    &:hover, &:focus, &:hover div, &:focus div {
+      background-color: $red !important;
+    }
+  }
+  .select2-default {
+      color: #fff !important;
+  }
+}
+
+.select2-drop {
+  padding: 10px 5px;
+  .select2-results {
+    font-size: 1.1em !important;
+    .select2-result-label {
+      padding: 5px 8px 6px;
+    }
+    .select2-highlighted {
+      background-color: $red;
+    }
+  }
+}
+
+
+// Other pages: about, help
+
+#page {
+  .container-narrow, .container-narrowest {
+    padding-top: 50px;
+    padding-bottom: 50px;
+  }
+}
+
+
+// COUNTRY EDITS
+
+.leaflet-label {
+  color: #fff !important;
+  background-color: $lightblue !important;
+  border-color: $lightblue !important;
+  border-radius: 0 !important;
+  font-weight: 200 !important;
+  font-family: $font-poppins !important;
+  padding: 3px 6px 1px !important;
+}
+
+.how-it-works {
+  img {
+    max-width: 70px;
+  }
+}
```

Comment:
So we should only override instead of copy pasting the `hurumap.scss`. That way if any stylistic updates happen on underlying hurumap they apply to PesaYetu.

Reply:
I tried overriding the styles as specified in the Wazimap docs, but it didn't work. I'll give it another go

Reply:
Ok. Not sure if `hurumap.scss` is following Wazimap docs as well so might want to check that too.